index.html

```diff
@@ -32,18 +32,16 @@
             <ul class="header__container__parhead__nav__ul">
               <li>
                 <div class="header__container__parhead__nav__ul__responsive">
-                  <img
-                    src="./static/imgs/Union@2x.png"
-                    alt="Union@2x"
-                    class="responsive header__container__parhead__nav__ul__responsive__respo"
-                  />
+                  <div class="bar bar__1"></div>
+                  <div class="bar bar__2"></div>
+                  <div class="bar bar__3"></div>
                 </div>
               </li>
               <li>
-                <ul class="iwq">
-                  <li><a href="#">Portoflio</a></li>
-                  <li><a href="#">About</a></li>
-                  <li><a href="#">Contact</a></li>
+                <ul class="iwq ">
+                  <li><a class="mobile-nav-links" herf="#">Portoflio</a></li>
+                  <li><a class="mobile-nav-links" href="#">About</a></li>
+                  <li><a class="mobile-nav-links" href="#">Contact</a></li>
```

Comment:
- [ ] As stated in the project requirement please make sure that When the user clicks (or taps) any of the mobile menu options, the correct part of the page is displayed and also correct the type spelling for "href" . Kindly work on it.

Reply:
-  Thanks 🙏for your review I have noticed what you told me to modify and I will work on it.😊